Jenkinsfile

```diff
@@ -523,6 +529,9 @@ pipeline {
                                 sh 'ARCH=x86_64 make publish-image-ibm'
                                 sh 'ARCH=aarch64 make publish-image-ibm'
                                 sh 'make publish-image-multi-ibm'
+                                //sh 'ARCH=x86_64 make publish-stress-test-image-ibm'
+                                //sh 'ARCH=aarch64 make publish-stress-test-image-ibm'
+                                //sh 'make publish-stress-test-image-multi-ibm'
```

Comment:
Are we waiting for this new repo to be created before merging this change?

Reply:
merge now, to start testing. change later when icr repo is ready.